fog-openstack.gemspec

```diff
@@ -31,4 +31,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubocop'
   spec.add_development_dependency 'rubyzip', '~> 0.9.9'
   spec.add_development_dependency 'shindo',  '~> 0.3'
+  spec.add_development_dependency 'pry'
```

Comment:
Hm, lets probably remove this, I haven't seen debugging tools added like this usually. You could just put them to your local gemfile